src/wazuh_db/wdb.c

```diff
@@ -214,6 +214,7 @@ static const char *SQL_STMT[] = {
     [WDB_STMT_TASK_CANCEL_PENDING_UPGRADE_TASKS] = "UPDATE TASKS SET STATUS = '" WM_TASK_STATUS_CANCELLED "', LAST_UPDATE_TIME = ? WHERE NODE = ? AND STATUS = '" WM_TASK_STATUS_PENDING "' AND (COMMAND = 'upgrade' OR COMMAND = 'upgrade_custom');",
     [WDB_STMT_PRAGMA_JOURNAL_WAL] = "PRAGMA journal_mode=WAL;",
     [WDB_STMT_PRAGMA_ENABLE_FOREIGN_KEYS] = "PRAGMA foreign_keys=ON;",
+    [WDB_STMT_PRAGMA_SYNCHRONOUS_1] = "PRAGMA synchronous = 1;",
```

Comment:
We should keep the format similar to the other configurations, eliminating the spaces between the = and the value.
`[WDB_STMT_PRAGMA_SYNCHRONOUS_1] = "PRAGMA synchronous=1;",`